Repaired seed file and changed rate per day

akilchomoko opened the pull request:
Correct the duplicate line and the rate per day calculation.

akilchomoko committed:
Repaired seed file and changed rate per day

gtm19 commented on db/schema.rb:
```diff
@@ -60,6 +60,7 @@
     t.string "address2"
     t.string "postcode"
     t.string "phonenumber"
+    t.integer "imdb_id"
```
@akilchomoko why does a user have an IMDB id?

akilchomoko replied:
It shouldn't. There should have been a migration that removed it from an earlier mistake.

akilchomoko committed:
Correctly removed the imdb_id from user_id table

akilchomoko commented:
Corrected the user_id table. Removed the incorrect imdb_id column. I thought I had done that migration earlier and it was all sorted but may have lost that change in the git learning process. Should be sorted now.

WaynePatrickDalton commented:
I just merged this to get us up-to-date.
